source/includes/orr-intro.rst

```diff
@@ -0,0 +1,6 @@
+Starting in {+c2c-beta-program-short+} 1.8, you can enable Oplog Rollover 
+Resilience (ORR). With ORR,  ``mongosync`` applies changes made on the source 
+cluster to the destination cluster during the initial sync. This adds resilience 
+to long-running operations, mitigates the risk of oplog rollover, and 
+significantly lowers the risk of restarting the sync.
```

Comment:
A couple suggestions on the phrasing for clarity:

For the `With ORR` sentence, the "during the initial sync" part is a bit ambiguous to me and may not fully describe the advantage of ORR to some readers. Even without ORR, mongosync still technically applies the events that occurred _during_ initial sync (without ORR, this application just happens _after_ the initial sync). I think we can make an emphasis that ORR allows mongosync to apply the events _concurrently_ with initial sync:
> With ORR, `mongosync` concurrently applies changes made on the source cluster to the destination cluster as initial sync is ongoing.

For the `This adds` sentence, the "long-running operations" phrase is a bit vague. It's not clear whether it's some user operation on the source, the sync itself, or something else. I'd rephrase it to something like:
> For source clusters with a high write rate, ORR significantly lowers the risk of oplog rollover during initial sync and needing to restart the sync.